optuna/study.py

```diff
@@ -358,6 +363,9 @@ def objective(trial):
                 By default, internal fields of :class:`~optuna.structs.FrozenTrial` are excluded
```

Comment:
Since `include_internal_fields`  has been removed, we also delete this description.

Reply:
Removed it in this commit (https://github.com/optuna/optuna/pull/775/commits/6d74394b25537b5eb845a5911e371d1a72b04d03). Thank you.